### ISSUE TITLE
Module displaying a description of the attributes

### ISSUE THREAD

**Issue by IGNF-Xavier** [geonetwork/geonetwork-ui#1007]
### Description

I would like to request adding a module that can display the description of the attributes of a vector dataset

### Motivation

For exemple when you have a daset with these attributes : 
![image](https://github.com/user-attachments/assets/7dbe33dd-7c51-40c0-abbd-e6dbc17171e4)

It is difficult to know what each attribute corresponds to.

### Inspiration and context

This request takes inspiration from Melodi from Insee catalog or Pomme ( an intranet at IGN)
https://catalogue-donnees.insee.fr/fr/catalogue/recherche/DS_ICA

**Comment by jahow** [comment]
Attribute descriptions can be found in the ISO metadata. They could also be fetched dynamically from a WFS service but that will probably be less rich.

**Comment by IGNF-Xavier** [comment]
ISO19110, so we can provide the link to the conceptual schema in the metadata and display it ?
The conceptual schema will be stored as an XML in a server or in a feature catalog in Geonetwork (https://docs.geonetwork-opensource.org/3.12/user-guide/associating-resources/linking-feature-catalog/)

**Comment by jahow** [comment]
Either that or ISO19115-3 records can contain the attributes description alongside all other dataset metadata in the same record.

**Comment by IGNF-Xavier** [comment]
Yes but i fear that the metadata will become a big file and difficult to update when the data is updated
Le 28 sept. 2024 10:11, Olivia Guyot a écrit :


Either that or ISO19115-3 records can contain the attributes description alongside all other dataset metadata in the same record.
